docker/dependencies/README.md

```diff
@@ -0,0 +1,26 @@
+# Overview
+
+Purpose of docker container, which is to be generated from this folder, is to test, if Iroha system can build, when all dependencies are installed into custom, separated directories.
+
+# How to build:
+1. Build docker image:
+    - ```cd $IROHA_ROOT/docker/dependencies```
+    - ```docker build --build-arg PARALLELISM=$DESIRED_THREADS_AMOUNT -t iroha-dpnd .```
+    - Now, you have container named "iroha-dpnd", containing all necessary dependencies in /opt/dependencies folder
+2. Get into the container:
+    - for the first time:
+        - run image: ```docker run -dit iroha-dpnd```
```

Comment:
I thought, that reusing the container without rebuilding it for test each time is useful. Dependencies rarely change, as I understand, so maybe just pulling new version of Iroha and then building it is enough?

Reply:
Yes, it will not be rebuilt, because it will be stored as image, and the image will not be removed after exit from container. Because image-container relationship can be seen as class-object, you can always create a new container from the previously built image. So it is safe to remove the container after usage, and later make a new one with `docker run --rm -it iroha-dpnd`.

Reply:
Ah, I understand now. Got a little confused about image-container relations